.github/workflows/python.yaml

```diff
@@ -36,7 +38,7 @@ jobs:
     - name: Cache conda packages
       uses: actions/cache@v2
       with:
-        key: ${{matrix.os}}-conda-pkgs-${{hashFiles('**/conda_recipe/meta.yaml')}}
+        key: ${{matrix.os}}-conda-pkgs-${{hashFiles('**/conda_recipe/meta.yaml', '**/*environment.yml')}}
```

Comment:
The standard name is actually just `environment.yaml`. Ideally it would be in the project root, so you can just run `conda env update` but given that we don't really use conda much (other than me) I don't think moving it to the root makes sense. I'll rename it to just `environment.yaml` though.

Reply:
Irritatingly, it appears that most conda stuff defaults to `.yaml`, but this one case uses `.yml`. So they are just different for now. Sigh. `environment.yml` is the standard. As is `meta.yaml`. Yuck.